packages/core/src/utils/ConfigurationLoader.ts

```diff
@@ -90,8 +90,15 @@ export class ConfigurationLoader {
   }
 
   static async getTsConfig(tsConfigPath: string): Promise<Dictionary> {
-    const json = await readFile(tsConfigPath);
-    return JSON.parse(stripJsonComments(json.toString()));
+    const tsConfigFile = await readFile(tsConfigPath);
+    const tsConfig = JSON.parse(stripJsonComments(tsConfigFile.toString()));
+    if (Object.keys(tsConfig).includes('extends')) {
+      const tsConfigFolderPath = dirname(tsConfigPath);
+      const tsConfigExtendsPath = isAbsolute(tsConfig.extends) ? tsConfig.extends : join(tsConfigFolderPath, tsConfig.extends);
+      const baseTsConfig = ConfigurationLoader.getTsConfig(tsConfigExtendsPath);
+      return Utils.merge(baseTsConfig, tsConfig);
+    }
+    return tsConfig;
```

Comment:
let's swap this, also add some spaces

```suggestion

    if (!Object.keys(tsConfig).includes('extends')) {
      return tsConfig;
    }

    const tsConfigFolderPath = dirname(tsConfigPath);
    const tsConfigExtendsPath = isAbsolute(tsConfig.extends) ? tsConfig.extends : join(tsConfigFolderPath, tsConfig.extends);
    const baseTsConfig = ConfigurationLoader.getTsConfig(tsConfigExtendsPath);

    return Utils.merge(baseTsConfig, tsConfig);
```